pkg/node/statestore.go

```diff
@@ -29,20 +29,29 @@ func InitStateStore(log logging.Logger, dataDir string) (ret storage.StateStorer
 }
 
 const overlayKey = "overlay"
+const secureOverlayKey = "non-mineable-overlay"
 
 // CheckOverlayWithStore checks the overlay is the same as stored in the statestore
 func CheckOverlayWithStore(overlay swarm.Address, storer storage.StateStorer) error {
```

Comment:
technically not related to this PR but eventually this method name should be updated since it's doing much more than just `Checking`.